pkg/genericapiserver/genericapiserver.go

```diff
@@ -269,6 +269,10 @@ type GenericAPIServer struct {
 	enableOpenAPISupport   bool
 	openAPIInfo            spec.Info
 	openAPIDefaultResponse spec.Response
+
+	// PostStartHooks is called in a go func after the server go func is launched, but not necessarily after
+	// the server is listening.  It may kill the process with a panic if it wishes to.
```

Comment:
> Seems an odd thing to encourage. Particularly when utilruntime.HandleCrash() isn't called in the initial Go statement. It this so PostStartHooks can bring down the API Server if they feel the need to?

Yes. I want to be sure they have the option.  I could be flexible on the expression of it though.